engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -350,13 +356,17 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
       url += WebServices.PATH;
     }
     catch (Exception ex) {
-      //LOG.error("Caught exception when retrieving web services info", ex);
+      LOG.error("Caught exception when retrieving web services info", ex);
```

Comment:
Please use debug. User can get details with -vvvv as is the case for everything else.